src/server.rs

```diff
@@ -172,17 +174,23 @@ async fn pair_handler(
 }
 
 pub async fn run() {
+    env::set_var("RUST_BACKTRACE", "1");
+    env::set_var("RUST_LOG", "tower_http=debug,ocean_chat=debug");
+    tracing_subscriber::fmt::init();
```

Comment:
I'd prefer to use something like this to configure the layer, it should make it possible to overwrite it with environment variables if needed 

```rust
tracing_subscriber::registry()
        .with(
            tracing_subscriber::EnvFilter::try_from_default_env().unwrap_or_else(|_| {
                "ocean_chat=debug,tower_http=debug,axum::rejection=trace".into()
            }),
        )
        .with(tracing_subscriber::fmt::layer())
        .init();
```

Reply:
nice catch.